node/subsystem-util/src/rolling_session_window.rs

```diff
@@ -354,13 +498,24 @@ async fn get_session_index_for_child(
 	}
 }
 
-async fn load_all_sessions(
+/// Attempts to extend db stored sessions with sessions missing between `start` and up to `end_inclusive`.
+/// If `allow_failure` is true, fetching errors are ignored until we get a first session, then
```

Comment:
`allow_failure` is a local variable - makes little sense to mention it in doc comment, better the actual condition.

Reply:
Yeah, I'll have to delete it, initially was a parameter.